src/components/molecules/search-results-list/search-results-list.scss

```diff
@@ -60,7 +61,7 @@
 	appearance: none;
 	background-color: white;
 	background-position: right 12px center;
-	background-size: 8px;
+	background-size: 0.5rem;
```

Comment:
rem or em?  I think we're using em everywhere else(for the time being).

Reply:
Ooof - no was left from trying to fix IE - will remove!